packages/frontend/core/src/components/affine/share-page-modal/share-menu/share-page.tsx

```diff
@@ -71,7 +77,7 @@ export const AffineSharePage = (props: ShareMenuProps) => {
     isSharedPage === null || sharedMode === null || baseUrl === null;
   const [showDisable, setShowDisable] = useState(false);
 
-  const currentDocMode = useLiveData(doc.primaryMode$);
+  const currentDocMode = useLiveData(editor.mode$);
 
   const mode = useMemo(() => {
```

Comment:
Can this be deleted?

Reply:
No, we need the current mode to determine whether to display `Copy Link to Selected Frame` option